src/GafferScene/SceneNode.cpp

```diff
@@ -55,6 +57,25 @@ GAFFER_GRAPHCOMPONENT_DEFINE_TYPE( SceneNode );
 
 size_t SceneNode::g_firstPlugIndex = 0;
 
+ValuePlug::CachePolicy childBoundsCachePolicy()
+{
+	ValuePlug::CachePolicy result = ValuePlug::CachePolicy::TaskIsolation;
+	if( const char *policy = getenv( "GAFFERSCENE_CHILDBOUNDS_CACHEPOLICY"  ))
+	{
+		if( !strcmp( policy, "TaskCollaboration" ) )
+		{
+			result = ValuePlug::CachePolicy::TaskCollaboration;
+		}
+		else if( strcmp( policy, "TaskIsolation" ) )
+		{
+			IECore::msg( IECore::Msg::Warning, "SceneNode", "Invalid value for GAFFERSCENE_CHILDBOUNDS_CACHEPOLICY. Must be TaskIsolation or TaskCollaboration." );
+		}
+	}
+	return result;
+}
```

Comment:
Do you plan to commit this as-is, or are you planning to hardcode this before merging?

Reply:
I'd like to keep it in for now. I don't think Collaboration had any particular benefits in Murray's production scenarios, but it's not hard to imagine a case where we'd want to try it again, and I'd like to be able to do that easily. Longer term I want to reduce the overhead of Collaboration and ditch Isolation as an option at which point we can remove this.